packaging/arch/snap-mgmt.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
```

Comment:
+1 on moving this to data and using for more than just Fedora.

Reply:
Yeah, I second that, it looks like this PR duplicates it, lets unify it instead :)